### DISCUSSION TITLE
Option to Change Outline Editor Width

### DISCUSSION THREAD

**Discussion by ba-sicheng** [vincode-io/Zavala#144] [Ideas]
Feature idea: Add an option to set the width of the Outline Editor.

Rationale: I often write paragraph-length topics and notes in my outlines. The view can get too wide to be readable on the desktop app, especially when working on a single document on its own window in full screen mode on a widescreen monitor.

Implementation possibilities: On macOS, the option could be added as a menu bar menu that would allow the user to choose between fit-to-window (the current configuration of the Outline Editor) and narrow (which would set a readable maximum limit the width of the Outline Editor). The option could also be added to the app preferences as a text field that would allow the user to input a maximum width (perhaps in pixels) for the Outline Editor.

PS: I love this app! Thank you for developing it! I've long been looking for a simple outliner app. (Opal used to be my preferred outliner, but it's no longer being actively developed.) John Gruber's original tweet immediately resonated with me when I saw it on my feed. I'm so glad you took action!

**Comment by vincode-io** [comment]
I think this is a good idea. I've created an issue for it and you can look for it in an future update.

**Comment by vincode-io** [comment]
I probably won't provide this as a preference though. Apple provides guidance on the readable width of text and that is probably what I'll restrict it to for everyone.

**Comment by vincode-io** [comment]
We have added this as a preference in the latest versions of Zavala.
